front/components/vaults/EditVaultStaticDatasourcesViews.tsx

```diff
@@ -9,7 +9,7 @@ import { useRouter } from "next/router";
 import React, { useCallback, useState } from "react";
 
 import VaultCreateWebsiteModal from "@app/components/vaults/VaultCreateWebsiteModal";
-import VaultFolderModal from "@app/components/vaults/VaultUpsertFolderModal";
+import VaultFolderModal from "@app/components/vaults/VaultFolderModal";
```

Comment:
This change and the one below is just because I renamed the file to match the component name (forgot to do it on my last PR 😬 )